.travis.yml

```diff
@@ -65,7 +65,7 @@ jobs:
       on:
         tags: true
 
-  - stage: master
+  - stage: trunk
```

Comment:
Was trying to think of a common term for master and prerelease branches. Open to suggestions.

Reply:
This stage name matches https://github.com/Workday/canvas-kit/pull/481/files#diff-354f30a63fb0907d4ad57269548329e3R26

Reply:
You'll have to change L27 to something like:
```
if: branch = master OR branch =~ /^prerelease\//
```

Reply:
Could go with my creative stage names: `branch`. I called the other stages `pull_request` and `tag`

Reply:
Ah - missed this. Thanks :)

Reply:
I think I'm going to stick with `trunk`. This stage should only run on "trunk" style branches that are being merged into (like `master` and `prerelease/v*`). If we used `branch` it makes it seem like it will run on all branches.